server/routes/serviceProviderReferrals/dashboardPresenter.ts

```diff
@@ -72,6 +80,8 @@ export default class DashboardPresenter {
   private readonly showAssignedCaseworkerColumn =
     this.dashboardType === 'All open cases' || this.dashboardType === 'Completed cases'
 
+  readonly isSearchable = this.dashboardType === 'All open cases' || this.dashboardType === 'Unassigned cases'
```

Comment:
Anything dependent on `this.dashboardType` could go to `dashboardDetails` to move away from branching in several places

Reply:
Sure will take care as part of next PR